__tests__/setup/waitForActivity.js

```diff
@@ -0,0 +1,5 @@
+import waitForObservable from './waitForObservable';
+
+export default function waitForBotRespond(directLine, predicate) {
```

Comment:
I would change the name of the function to `waitForBotResponse` or `waitForBotToRespond`